addons/fluentbit/1.5.x/fluentbit-2.yaml

```
@@ -0,0 +1,192 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: fluentbit
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: fluentbit
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.2-1"
```

Updating addon revisions is currently (unfortunately) a manual process. We'll need to adjust accordingly with this new fluentbit revision:

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.2-2"
```